### PR TITLE
new inputs validation

### PR DESCRIPTION
**Pull request by eduardoccmelo**
- add new inputs validation
- manipulate better trip dates
-  change some css

### CODE REVIEW

File: src/components/Trip.js [hunk 1]
```diff
@@ -98,7 +106,9 @@ export default function Trip() {
             <div className="tripDetailsFieldContent">
               <i className="far fa-calendar-alt"></i>
               {singleTrip.checkoutDate.slice(8, 10)}.
-              {singleTrip.checkoutDate.slice(5, 7)}
+              {new Date(singleTrip.checkoutDate).toLocaleString("default", {
+                month: "short",
+              })}
```
Comment (ManuelRemer) [inline src/components/Trip.js:111]: Just to leave a comment: you are using almost the same div with almost the same functions in it several times. You could make another component of it (I now, you are refactoring right now) . And give it the className of <i> and singleTrip key as a prop.

File: src/components/styles/MyTrips.css [hunk 2]
```diff
@@ -240,16 +255,16 @@ hr {
   }
 
   .tripDates {
-    right: 32px;
-    top: 8px;
-    font-size: 16px;
-    line-height: 1.3;
+    right: 29.5px;
```
Comment (ManuelRemer) [inline src/components/styles/MyTrips.css:258]: looks like perfectionism

**Review by ManuelRemer — COMMENTED** [review]
Everything is working fine. Just the go up arrow might be confusing if you haven't scrolled down before. Cool.

File: src/components/Trip.js (hunk 1 above)
Comment (eduardoccmelo) [reply src/components/Trip.js:111]: Thanks Manuel

**Pull request merged**